packages/react-portal/.babelrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "plugins": ["module:@fluentui/babel-make-styles", "annotate-pure-calls", "@babel/transform-react-pure-annotations"]
+  "plugins": ["annotate-pure-calls", "@babel/transform-react-pure-annotations"]
```

Comment:
hmm going forward we should revisit the approach here. currently storybook is being build and published only via `react-components` suite, but that might change - if that will happen we'll need to provide these transforms also for this scenario - package that include stories  that use griffel although griffel is not used in package source implementation.

Reply:
https://github.com/microsoft/fluentui/issues/21452